package.json

```diff
@@ -25,6 +25,7 @@
     "@saleor/app-sdk": "0.51.0",
     "@saleor/macaw-ui": "1.1.10",
     "@urql/exchange-auth": "^1.0.0",
+    "@vercel/kv": "^3.0.0",
```

Comment:
this is wrong, vercel kv should not be required to run, unless you chose to use kv in APL.

Does it work if you use eg FileAPL?

Reply:
I commented out Vercel KV so it won't be included by default